conformance/utils/traffic/traffic.go

```diff
@@ -60,6 +83,7 @@ func BuildExpectedHTTPResponse(
 	return resp
 }
 
+// Deprecated: please use MakeRequestAndExpectSuccessV2 instead.
```

Comment:
Can you create an issue to track this? (If you already have, please link it here).